docs_nnx/api_reference/flax.nnx/spmd.rst

```diff
@@ -7,4 +7,3 @@ spmd
 .. autofunction:: get_partition_spec
 .. autofunction:: get_named_sharding
 .. autofunction:: with_partitioning
-.. autofunction:: with_sharding_constraint
```

Comment:
This fixes a warning on docs build as the function was removed in https://github.com/google/flax/pull/4893